src/controller/cve.controller/index.js

```diff
@@ -249,7 +225,7 @@ router.put('/cve/:id',
         <p>User must belong to an organization with the <b>Secretariat</b> role</p>
         <h2>Expected Behavior</h2>
         <p><b>Secretariat:</b> Updates a CVE record for any organization</p>"
-  #swagger.parameters['id'] = { description: 'The CVE ID for which the record is being updated.' }
+  #swagger.parameters['id'] = { description: 'The CVE ID for the record being updated' }
   #swagger.parameters['$ref'] = [
```

Comment:
PUT /cve/:id for Secretariat will have the same request body as the POST - it replaces the record with the new body